unity-renderer/Assets/UIComponents/Scripts/Components/Dropdown/DropdownComponentView.cs

```diff
@@ -0,0 +1,465 @@
+using DCL.Helpers;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public interface IDropdownComponentView
```

Comment:
In the case of the UIComponents views, I prefer to keep the interfaces inside the same file with its own implementation because in this case they'll only be implemented once and it alse eases the navigation through the code. All the rest of UIComponents views are implemented in the same way. Anyway, as far as I know, we didn't define a standard for that, so maybe it could be a good point to discuss with the rest of the team 👍

Reply:
i think we should have different files for each individual class, interface, struct, enum unless are internal (class SomeClass { struct InternalStruct {} }).
Makes a more organized project structure, mitigates some merge conflicts and is prepared for new interface implementations. Since we have all ui components like this, perhaps we could do it in a different pr